src/cpp/processquit.cpp

```diff
@@ -213,6 +221,7 @@ public slots:
 #endif
 	}
 
+public:
 	void sig_activated(int)
 	{
```

Comment:
Let's move this function up to the earlier public section rather than having two public sections.